docs/environment-variables.md

```diff
@@ -139,7 +139,8 @@ They are only usable within workers. Please set the environment variables in the
 | `GPUSTACK_RUNTIME_DEPLOY_MIRRORED_NAME`                            | The name of the deployer.                                                                                 | (empty)                                         |
 | `GPUSTACK_RUNTIME_DEPLOY_MIRRORED_DEPLOYMENT_IGNORE_ENVIRONMENTS`  | Environment variable names to ignore during mirrored deployment.                                          | (empty)                                         |
 | `GPUSTACK_RUNTIME_DEPLOY_MIRRORED_DEPLOYMENT_IGNORE_VOLUMES`       | Volume mount destinations to ignore during mirrored deployment.                                           | (empty)                                         |
-| `GPUSTACK_RUNTIME_DEPLOY_CORRECT_RUNNER_IMAGE`                     | Correct the gpustack-runner image by rendering it with the host's detection.                              | `1`                                             |
+| `GPUSTACK_RUNTIME_DEPLOY_CORRECT_RUNNER_IMAGE`                     | Correct the gpustack-runner image by rendering it with the host's detection.                              | `0`                                             |
```

Comment:
Remove environment variables that have no use case.

Reply:
Updated